### DISCUSSION TITLE
Ability to edit the color of lottie animations, in elementor🔥

### DISCUSSION THREAD

**Discussion by 79ho3ein** [elementor/elementor#15462] [Feature Request]
hello
Is it possible for us to change the colors of these animations in elementor?
If such a feature is added to this widget so that we can change the colors in each animation using elementor, this would be great.
Currently, we can only change the color of any animation on lottiefiles.com : https://prnt.sc/16amvjy
If you add the ability to change the color of these animations to the lottie widget, it will be a great achievement.
The lottie widget is one of the best elementor widgets. Which, of course, will be more productive in the future.

Thank you for your efforts

**Comment by 79ho3ein** [comment]
hi @Aviad-Herman
Is it possible to implement this proposal?

thanks you

**Comment by Aviad-Herman** [comment]
Hey, I'm converting this to a feature request discussion but we don't have plans for this anytime soon
